seqr/views/apis/anvil_workspace_api.py

```diff
@@ -174,8 +178,16 @@ def create_project_from_workspace(request, namespace, name):
         logger.error('Uploading sample IDs to Google Storage failed. Errors: {}'.format(str(ee)), request.user,
                      detail=sample_ids)
 
-    # Send a slack message to the slack channel
-    _send_load_data_slack_msg(project, ids_path, data_path, request_json['sampleType'], request.user)
+    # use airflow api to trigger AnVIL dags
+    try:
+        _trigger_anvil_dags(project, data_path, request_json)
+        # Send a slack message to the slack channel
+        _send_load_data_slack_msg(project, ids_path, data_path, request_json['sampleType'], request.user)
+    except Exception as e:
+        logger.error(e)
```

Comment:
the `logger` class here requires a secondary argument with the user - this needs to be `logger.error(e, request.user)`